Lib/sre_constants.py

```diff
@@ -21,6 +21,17 @@
 # should this really be here?
 
 class error(Exception):
+    '''Exception raised for invalid regular expressions.
+
+    The error instance has the following additional attributes:
+
+    msg: The unformatted error message
+    pattern: The regular expression pattern
+    pos: The index of pattern where compilation failed
```

Comment:
'in' instead of 'of' sounds better to me.
If this is also raised during matching, what would this be?